### PR TITLE
Add the book review for Intro to Service Design.

### PR DESCRIPTION
**Pull request by nickcharlton**
Preview page: https://deploy-preview-32--nickcharlton-net.netlify.com/posts/book-review-introduction-to-service-design.html

### CODE REVIEW

File: _posts/2017-11-25-book-review-introduction-to-service-design.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Book Review: How To Have Your Cake And Eat It Too: An Introduction to Service Design"
+published: 2017-11-25 23-27-41 +00:00
+tags: book service-design
+---
+
+I am a software developer, but also a business consultant. Every year I work
+with a bunch of small companies, founders and those who are running teams in
+much larger organisations. These are people who /do/care about the quality of
```
Comment (danielgroves) [inline _posts/2017-11-25-book-review-introduction-to-service-design.md]: /do/care?

File: _posts/2017-11-25-book-review-introduction-to-service-design.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* What journey does a customer take in using your service?
+* What are they trying to achieve?
+* What are their pain points?
+
+These can be answered by building personas, drawing out a customer journey map
+and then doing customer interviews to fill in the gaps, respectively.
+
+The consultant is merely a product of the tools in their toolbox. I’ve
+participated in several, and since run, a design sprint. I’ve built products
+using various flavours (and in various levels of dysfunction) of agile. But I’d
```
Comment (danielgroves) [inline _posts/2017-11-25-book-review-introduction-to-service-design.md]: I'd => I've?

File: _posts/2017-11-25-book-review-introduction-to-service-design.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+been hunting for something out of those which is missing, and a step into this
+“service design” thing I’d been hearing about regularly.
+
+J. Margus Klaar presents us with brief overview of all of the above. At 106
+   pages, it’s short enough to read and digest in a couple of hours and present
+   you with another collection of tools to have at your disposal.
+
+They discuss how consumer actions have changed over the past fifty years and
+changes of the world which are causing peoples’ behaviour to change. They
```
Comment (danielgroves) [inline _posts/2017-11-25-book-review-introduction-to-service-design.md]: "Change of the word" doesn't quite make sense to me, did you mean "to the world"?

File: _posts/2017-11-25-book-review-introduction-to-service-design.md (hunk 1 above)
Comment (nickcharlton) [reply _posts/2017-11-25-book-review-introduction-to-service-design.md]: Missing a space.

File: _posts/2017-11-25-book-review-introduction-to-service-design.md (hunk 2 above)
Comment (nickcharlton) [reply _posts/2017-11-25-book-review-introduction-to-service-design.md]: Hmm, but then it's "I've" three times in a row. I think this bit should probably be rephrased entirely.

**Pull request merged**